api/features/views.py

```diff
@@ -588,6 +588,26 @@ def get_queryset(self):
             raise NotFound("Environment not found.")
 
 
+@swagger_auto_schema(
+    responses={200: WritableNestedFeatureStateSerializer()}, method="get"
+)
+@api_view(["GET"])
+@permission_classes([IsAuthenticated | HasMasterAPIKey])
+def get_feature_state_by_uuid(request, uuid):
+    if getattr(request, "master_api_key", None):
+        accessible_projects = request.master_api_key.organisation.projects.all()
+    else:
+        accessible_projects = request.user.get_permitted_projects(["VIEW_PROJECT"])
+    qs = (
+        FeatureState.objects.filter(feature__project__in=accessible_projects)
+        .select_related("feature_state_value")
+        .prefetch_related("multivariate_feature_state_values")
```

Comment:
If this is a request for a single object, I don't think the prefetch related actually adds anything does it?